internal/prober/multihttp/script.go

```diff
@@ -34,12 +59,29 @@ func buildUrl(req *sm.MultiHttpEntryRequest) string {
 			buf.WriteString(url.QueryEscape(field.Value))
 		}
 	}
-
 	u.RawQuery = buf.String()
 
 	return template.JSEscapeString(u.String())
 }
 
+func buildQueryParams(req *sm.MultiHttpEntryRequest) string {
+	var buf strings.Builder
+	for i, field := range req.QueryFields {
+		if i > 0 {
+			buf.WriteRune('\t')
+		}
+		buf.WriteString("url.searchParams.append(`")
```

Comment:
the string `url` should be an argument to this function, so that the expectation that there's a variable called `url` in the context where this is being called is not so strong.

```
func buildQueryParams(url string, req ...)
```